library/objective-c/EnvoyHTTPFilterCallbacksImpl.h

```diff
@@ -0,0 +1,12 @@
+#import <Foundation/Foundation.h>
```

Comment:
So multiple objc header files work now? If so, maybe we can (in a separate PR) look into splitting up the EnvoyEngine.h file?

Reply:
Not quite. Private headers can be used either by listing them as `srcs` or by moving them to a separate library target. But anything exposed to Swift still needs to be in the single header (because that becomes the bridge header and we hit a clang error with transitive imports).